core/api/src/migrations/000053-removeCompoundindex.ts

```diff
@@ -0,0 +1,68 @@
+module.exports = {
```

Comment:
@bleonard I'm pretty sad about this whole situation.  Have any alternative ideas?

Reply:
because it's not possible in SQLite?

Reply:
I wonder if it could only do the first one in SQL lite somehow but we could still optimize postgres. The postgres instances are much better, so I'd want to optimize them a bit. Some `dynamicIndex` method that did something else if in SQLlite.

Reply:
That's an interesting idea - We could:
* Always include model-level (in code) validations which would work for PG and SQLite
* If the `config.seqeulize.dialect === 'postgres'`, we could also add compound index, to be extra safe

Reply:
Sounds good. It will be super clean if we wrap that in a method that is more optional and no-op on sqlite.